File: 8kyu-Convert-Boolean-Values-to-Strings.js
// Complete the method that takes a boolean value and return a "Yes" string for true, or a "No" string for false.

const { assert } = require("chai")

describe("Basic tests", () => {
  it("Testing for basic tests", () => {
    assert.strictEqual(boolToWord(true), 'Yes')
    assert.strictEqual(boolToWord(false), 'No')
    });
  });

// my code

  function boolToWord( bool ){
    return (bool == true) ? 'Yes' : 'No'
  }

File: CodeWars/.cs/8kyu-ConvertBooleanValuestoStrings.cs
// Complete the method that takes a boolean value and return a "Yes" string for true, or a "No" string for false.

using System;
using NUnit.Framework;

[TestFixture]
public class  boolToWordTest {

  [Test]
  public void boolToWordReturned1() {
    Assert.AreEqual("Yes", Kata.boolToWord(true));
    Console.WriteLine("Expected Yes");
  }

  [Test]
  public void boolToWordReturned2() {
    Assert.AreEqual("No", Kata.boolToWord(false));
    Console.WriteLine("Expected No");
  }

  [Test]
  public void boolToWordReturned3() {
    Assert.AreEqual("Yes", Kata.boolToWord(true));
    Console.WriteLine("Expected Yes");
  }
}

// my code

public static class Kata
{
  public static string boolToWord(bool word) => (word) ? "Yes" : "No";
}